ckan/lib/helpers.py

```diff
@@ -1010,6 +1007,14 @@ def _range(self, regexp_match):
         return re.sub(current_page_span, current_page_link, html)
 
 
+def get_display_timezone():
```

Comment:
missing a docstring. what sort of values are returned from this function? Can we rely on this helper returning the time zone the user will see by default (no, so you should warn about that)

Would it be more useful to return the timezone object here instead of the name? That would be more flexible, but there's no other code that uses this method so it's hard to judge.